source/aggregation-pipeline-builder.txt

```diff
@@ -205,8 +205,12 @@ your chosen view mode:
             On the :guilabel:`Stage Wizard` card, select a stage use 
             case for the first stage of the pipeline. You can click the stage 
             card to add it to the end of your pipeline or drag it to your 
-            preferred position. |compass-short| populates the stage card with a 
-            form that corresponds with the selected aggregation pipeline stage.
+            preferred position. 
+            
+            You can also filter the use cases by searching for the associated 
```

Comment:
[suggestion]

I'm not sure we need the word "also" here. I might be reading this incorrectly given that I'm not actually looking at the stage wizard tool, but it seems like the preceding paragraph is talking about how to add a stage to the pipeline, and this paragraph is talking about searching for a stage to add to the pipeline.

Reply:
Can take it out! 
Yeah if you're looking at the stage wizard tool, there's a search bar that can help users filter the results. Like you mentioned in the below comment, I'll probably move it up to the paragraph above it — that was something I was going back and forth on.